packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -40,6 +42,40 @@ export interface IProject extends IResource, iam.IGrantable, ec2.IConnectable {
 
   addToRolePolicy(policyStatement: iam.PolicyStatement): void;
 
+  /**
+   * Adds permissions to this project's role
+   * (if it has a role assigned, i.e. is not an imported project)
+   * to create test report groups with the name starting with the name of the project.
+   * Used when you specified a simple name
+   * (as opposed to an ARN)
+   * of a report group in your buildspec.
+   *
+   * @returns true if the permissions were added,
+   *   false if either this project is imported,
+   *   or its role is imported in an immutable way,
+   *   and so the call had no effect
+   *
+   * @see https://docs.aws.amazon.com/codebuild/latest/userguide/test-report-group-naming.html
+   */
+  createsReportGroups(): boolean;
```

Comment:
I guess don't do this. Follow the grant pattern and have it return an `iam.Grant`?

Reply:
Done.